src/mono/wasm/runtime/exports.ts

```diff
@@ -288,4 +310,10 @@ export interface BINDING {
     bind_static_method: typeof mono_bind_static_method,
     call_assembly_entry_point: typeof mono_call_assembly_entry_point,
     unbox_mono_obj: typeof unbox_mono_obj
+}
+interface DotNetExports {
+    MONO: MONO,
+    BINDING: BINDING,
+    INTERNAL: any,
+    Module: any
```

Comment:
Is there a specific reason why Module and INTERNAL don't have specific types here? I thought we had interfaces for them. I don't necessarily think they need it though

Reply:
This will be visible in `dotnet.d.ts`, which is our published API. `INTERNAL` and `Module` are not part of that API contract.